hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/model/translator.kt

```diff
@@ -108,14 +108,12 @@ fun Prisoner.translate(existingPrisoner: Prisoner? = null, ob: OffenderBooking,
     ob.sentenceDetail?.conditionalReleaseOverrideDate ?: ob.sentenceDetail?.conditionalReleaseDate
   this.actualParoleDate = ob.sentenceDetail?.actualParoleDate
 
-  // get the most serious offence for this booking
-  this.mostSeriousOffence =
-    ob.offenceHistory?.firstOrNull { off -> off.mostSerious && off.bookingId == ob.bookingId }?.offenceDescription
```

Comment:
Moved this logic to the prison-api endpoint to make space for a future change to return the entire offence history.